Helper/Config.php

```diff
@@ -189,6 +197,24 @@ public function getAccessToken(): ?string
         return $this->encryptor->decrypt($this->scopeConfig->getValue(self::ACCESS_TOKEN_CONFIG_KEY));
     }
 
+    /**
+     * Retrieves the Merchant ID associated with this platform install
+     */
+    public function getMerchantId(): ?string
+    {
+        return $this->encryptor->decrypt($this->scopeConfig->getValue(self::MERCHANT_ID_CONFIG_KEY));
+    }
+
+    /**
+     * Generates the Merchant ID associated with this platform install
+     */
+    public function generateMerchantId(): string
+    {
+        $merchantId = uniqid(rand(), true);
```

Comment:
PHP & Magento's ability to generate a UUID is extremely poorly implemented without additional libraries. However, the minor use-case of this does not seem to warrant an additional library. Given this, we are returning a unique ID based off of the random value (based on the current time in microseconds) with the additional encryption flag set to true. This results in a value such as:
`8225347095f188df3e659e8.90316476`